### ISSUE TITLE
Component is not compatible with JAWS screen reader

### ISSUE THREAD

**Issue by johannesh2** [vaadin/vaadin-cookie-consent#40]
Results from #12 

JAWS on Win 7 (both with IE 11 and Chrome)
3.1. when the page loads: in Chrome JAWS does not announce the cookie consent popup, but in IE11 that works OK.
3.2. when the focus is moved to the first focusable element in the popup (the 'learn more' link): JAWS does not read the title nor the content of the dialog. Thus, in Chrome the consent dialog text is never read out loud.
